config/settings.sample.yml

```diff
@@ -4,6 +4,7 @@ development: &non_production_settings
   :authoring_site_url: //authoring.staging.concord.org
   :codap_url: http://codap.concord.org/releases/latest/
   :log_manager_data_interactive_url: //cc-log-manager.herokuapp.com/di
+  :elastic_search_endpoint_url: http://search-has-portal-prod-xruhhhyiv2fugtujtzbgfq7bem.us-east-1.es.amazonaws.com
```

Comment:
Sounds good, but just to understand for the future -- is there a reason why some urls should be in environment variables, and others (like the codap url etc above) are in settings?